inference-engine/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ set(TARGET_NAME funcSharedTests)
 list(APPEND EXPORT_DEPENDENCIES
         funcTestUtils
         ngraphFunctions
+        MKLDNNPlugin
+        MultiDevicePlugin
+        HeteroPlugin
```

Comment:
it's not export dependencies. Need to add to `addIeTarget` to the section `DEPENDENCIES`

Reply:
Done